.github/workflows/update-search.yml

```
@@ -0,0 +1,55 @@
+    - cron: '0 4 * * *'  # Run daily at 4:00 UTC
+  workflow_dispatch:  # Allow manual trigger
+
+# Add permissions needed for the workflow
+permissions:
+  contents: write    # This allows the action to commit and push changes
+
+jobs:
+  update-search:
+    runs-on: ubuntu-latest
+    
+    steps:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 19: did not find expected '-' indicator

(syntax-check)